### PR TITLE
javascript3-week3/NicoQ

### PR DESCRIPTION
**Pull request by lnknfan90**
Thanks for the review  :)

### CODE REVIEW

File: 3_javascript/javascript3/week3/screenshot_app/script.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+  function deleteSelectedScreenshot() {
+    const selectedScreenshotNumber = parseInt(selectScreenshot.value);
+
+    if (!isNaN(selectedScreenshotNumber)) {
+      const selectedIndex = selectedScreenshotNumber - 1;
+
+      fetchScreenshots()
+        .then((data) => {
+          if (selectedIndex >= 0 && selectedIndex < data.length) {
```
Comment (ddobby94) [inline 3_javascript/javascript3/week3/screenshot_app/script.js:131]: There is an easier way to check if the index is available in the array:
```js
if (data[selectedIndex] !== undefined) {
  // ....
}
```

File: 3_javascript/javascript3/week3/screenshot_app/script.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        })
+        .catch((error) => console.error("Error", error))
+        .finally(() => {
+            loadingIndicator.style.display = "none";
+          });
+    });
+  }
+
+  function getScreenshotURL(URL) {
+    return URL;
```
Comment (ddobby94) [inline 3_javascript/javascript3/week3/screenshot_app/script.js:54]: It seems like this function doesn't do anything, meaning you can just delete the function and use the `URL` variable by itself.

Comment (ddobby94) [reply 3_javascript/javascript3/week3/screenshot_app/script.js:54]: So at line 27 it would be:
`const screenshotURL = URL;`

**Review by ddobby94 — COMMENTED** [review]
Hi,

I'm David Dobiasz a new mentor in HYF. If you have any question regarding any of my comments feel free to reach out to me on Slack, I am very happy to talk about them! Programming can be difficult and overwhelming with all the little things at first... So really reach out to me if you are in doubt about anything.

You did good on this homework I really didn't have any big comments, just some smaller things here and there :)

File: 3_javascript/javascript3/week3/screenshot_app/script.js (hunk 1 above)
Comment (lnknfan90) [reply 3_javascript/javascript3/week3/screenshot_app/script.js:131]: Nice!

**Pull request closed without merging**